UID2.Client.nuspec

```diff
@@ -2,7 +2,7 @@
 <package>
   <metadata>
     <id>UID2.Client</id>
-    <version>5.3.0</version>
+    <version>5.3.1</version>
```

Comment:
whatever next release, it should be 5.4.0, your changes aren't a patch so shouldn't be 5.3.1?